src/Pact/Native.hs

```diff
@@ -478,7 +478,7 @@ chainDataDef = defRNative "chain-data" chainData
       PublicData{..} <- view eePublicData
 
       let PublicMeta{..} = _pdPublicMeta
-          toTime = toTerm . posixSecondsToUTCTime . fromMicroseconds
+          toTime = toTerm . fromPosix . fromTimestampMicros
```

Comment:
Can we get a `timeToMicros` (this) and `microsToTime` (see below) that gets rid of the POSIX noise in userland please